src/draw/lv_image_decoder.h

```diff
@@ -49,6 +49,22 @@ struct _lv_image_decoder_dsc_t;
 struct _lv_image_decoder_t;
 struct _lv_cache_entry_t;
 
+/**
+ * Image decoder args.
+ * It determines how to decoder an image, e.g. whether to premultiply the alpha or not.
+ * It should be passed to lv_img_decoder_open() function. If NULL is provided, default
+ * args are used.
+ *
+ * Default args:
+ * all field are zero or false.
+ */
+typedef struct _lv_image_decoder_args_t {
```

Comment:
It should be highlighted that it's not sure that the request will be full filled. E.g. a decoder might not support pre multiplied or stride aligned decoding.